src/main/kotlin/racingcar/infrastructure/ConsoleOutput.kt

```diff
@@ -0,0 +1,31 @@
+package racingcar.infrastructure
+
+import racingcar.domain.Car
+
+class ConsoleOutput : Output {
```

Comment:
이런식으로 Impl를 통해 추상체를 구현하고 있음을 나타내주면 좀 더 명확해질 거 같습니다!
```suggestion
class ConsoleOutputImpl : Output {
```

Reply:
저는 이미 타입 명시가 되있고 함수들도 override 키워드가 있어 괜찮아 보여요!